easybuild/easyconfigs/p/Pyomo/Pyomo-5.5.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'Pyomo'
+version = '5.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pyomo.org/'
+description = """ Pyomo is a Python-based open-source software package that supports a diverse set of optimization
+ capabilities for formulating and analyzing optimization models. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+exts_defaultclass = 'PythonPackage'
+
+exts_download_dep_fail = True
```

Comment:
@smoors Please also set `use_pip = True`, you can do both at once with:

```python
exts_default_options = {
  'download_dep_fail': True,
  'use_pip': True,
}

Reply:
note to self: don't blindly copy code from @boegel without checking for style :)